portal-api/src/v1/gef/models/application.js

```diff
@@ -68,6 +68,7 @@ class Application {
         'facilitiesUpdated',
         'ukefDecisionAccepted',
         'portalActivities',
+        'ukefDecision'
```

Comment:
Is this decision, only added by TFM? I.e, GEF should not be able to change or modify this itself right?

I know it's the api model and TFM will end up calling this - maybe ideally "fields that GEF shouldn't change itself" should be a seperate endpoint/model .... food for thought.

Could we add a comment in the code to mention "gef shouldn't update this itself" kind of thing?

Reply:
yes, it's only generated from tfm-api. this would/should only be used for api-tests/e2e
I've added a comment in the code.

we can look into separate endpoints as well. it wouldn't be part of this PR though